modules/cucumber-reportportal-formatter.js

```diff
@@ -366,10 +411,16 @@ const createRPFormatterClass = (config) => {
           this.contextState.context.stepStatus === STATUSES.FAILED)
       ) {
         switch (event.media.type) {
-          case 'rp/testCaseId': {
+          case RP_EVENTS.TEST_CASE_ID: {
+            const dataObj = utils.getJSON(event.data);
+            const itemId = this.contextState.context.stepId;
```

Comment:
Maybe you should move `dataObj and itemId` outside of the switch. Since I see that they are used in two blocks, and probably when adding a new one, they will also participate

Reply:
Good catch. Fixed